Dockerfile

```diff
@@ -1,23 +1,17 @@
-FROM openmicroscopy/omero-ssh-c7:0.1.0
+FROM openmicroscopy/omero-ssh
```

Comment:
If needed, I can look at versioning `omero-ssh` as well. But it'll need a new name.